generators/app/conf.js

```diff
@@ -142,7 +142,7 @@ module.exports = function webpackConf(props) {
     conf.module.loaders.push({
       test: lit`/\\.js$/`,
       exclude: lit`/(node_modules|.*\\.spec\\.js)/`,
-      loader: 'isparta-instrumenter'
+      loader: 'isparta-loader'
```

Comment:
If I may, you don't need to add the suffix "loader" `loader: 'isparta'` will do the same